### DISCUSSION TITLE
The custom font used for the GitHub app breaks hex numbers (0x0) with a ligature

### DISCUSSION THREAD

**Discussion by Kwpolska** [community/community#30590] [Mobile]
The font used by the GitHub app has a ligature for `x` in `0x0` that centers the `x`, like a multiplication sign `×`. This means hex numbers look out of place and might be confused with multiplication. Also, the combinations 0x9 and 0xA look weird next to one another.

Screenshot:

![Screenshot_20220823-224207_GitHub.png](https://user-images.githubusercontent.com/327323/186261506-e1219369-f1ac-4ae4-8ce7-6359f778dcfc.png)

**Comment by taki-on** [comment]
Thank you for your report. The GitHub Android app uses the system default font. We tried to reproduce the issue on our devices, but we could not. Would you mind sharing more details about your device and OS version?

**Reply by Kwpolska** [reply]
The device is running Android 12, the GitHub app is at version 1.76.0, and the screenshot in the issue comes from the app. This is a Samsung device; I do not use the Samsung custom fonts feature, and this doesn’t happen with Roboto or Google Sans/Product Sans.
